examples/linux/block_device/mirror/main.cpp

```diff
@@ -0,0 +1,90 @@
+#include <xpcc/architecture.hpp>
+#include <xpcc/debug/logger.hpp>
+
+#include <xpcc/driver/storage/block_device_file.hpp>
+#include <xpcc/driver/storage/block_device_mirror.hpp>
+
+// Set the log level
+#undef	XPCC_LOG_LEVEL
+#define	XPCC_LOG_LEVEL xpcc::log::INFO
+
+void printMemoryContent(uint8_t* address, std::size_t size) {
+	for (std::size_t i = 0; i < size; i++) {
+		XPCC_LOG_INFO.printf("%x", address[i]);
+	}
+}
+
+struct FilenameA {
+	static constexpr const char* name = "testA.bin";
```

Comment:
Did you mean to commit the `examples/linux/block_device/mirror/testA.bin` and `testB.bin` files below?

Reply:
Yes, because otherwise the `test*.bin` files have to be created manually by the user (or CI) before running the example.
The is no nice way to create the files from C++.

Idea: Create the file in the SConstruct:
```python
# path to the xpcc root directory
xpccpath = '../../../..'
# create empty `~test.bin` file (if it does not exist)
open('~test.bin', 'a').close()
# execute the common SConstruct file
exec(compile(open(xpccpath + '/scons/SConstruct', "rb").read(), xpccpath + '/scons/SConstruct', 'exec'))
```
(Filename changed to `~test.bin` so that it is ignored by [gitignore](https://github.com/roboterclubaachen/xpcc/blob/develop/.gitignore).)